sktime/datatypes/_convert.py

```diff
@@ -128,16 +143,22 @@ def convert(obj, from_type: str, to_type: str, as_scitype: str = None, store=Non
 
 
 # conversion based on queriable type to specified target
-def convert_to(obj, to_type: str, as_scitype: str = None, store=None):
+def convert_to(
+    obj,
+    to_type: Union[Mtypes, List[str], List[Mtypes]],
```

Comment:
`List[str]` is unnecessary, since `Mtype` can also be a `str`, no?

Reply:
Good spot